app/characterization_services/geo_characterization_service.rb

```diff
@@ -35,18 +35,18 @@ def parent
   end
 
   def external_metadata_service
-    @external_metadata_service ||= ExternalMetadataCharacterizationService.new(file_set: file_set, persister: persister)
+    ExternalMetadataCharacterizationService.new(file_set: file_set, persister: persister)
```

Comment:
Why were these memoized/unmemoized?  If they are only being constructed to have one method call, should `ExternalMetadataCharacterizationService` be refactored to just have a class method in order to avoid the construction of objects?